CMakeLists.txt

```diff
@@ -6,8 +6,14 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 
 include(Colors)
 
-option( USEQTHREAD "Use QThread threading library" false )
 find_package( QThreads )
+find_package( UT )
```

Comment:
We probably need to make sure we add documentation for this one, as in what it is, where to get it, and how to compile + limitations. There's definitely advantages over QThreads.